androidApp/src/main/java/com/EENX15_22_17/digital_journal/android/ui/Utils.kt

```diff
@@ -0,0 +1,11 @@
+import androidx.compose.ui.Modifier
+
+/**
+ * Applies the supplied [modifier] to the chain only if [condition] is `true`.
+ */
+fun Modifier.modifyIf(condition: Boolean, modifier: Modifier): Modifier =
```

Comment:
is this now a extensionfunction for Modifier?

Reply:
Yep, it was possible to use `.then()` and merging with a new conditional modifier, but this reads better I think :)